README.txt

```diff
@@ -399,6 +404,23 @@ popd
 echo Failed.
 exit /b 1
 
+Windows Installer using WIX Toolset
+-----------------------------------
+
+Requirements: 
+	* Visual Studio 2017
+	* Wix Toolset
+
+In Visual Studio 2017 Installer select "optional Features" -> "Wix Toolset"
+
+Build the binaries like described above, but make sure you unselect "RUN_IN_PLACE".
+"RUN_IN_PLACE" builds are not designed for installations and leads to errors
```

Comment:
this is also confusing because creating RUN_IN_PLACE=1 installer builds is impossible (prevented by the build system)